mmv1/products/datastream/terraform.yaml

```diff
@@ -164,6 +176,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           stream_cmek: "kms-name"
         test_vars_overrides:
           deletion_protection: "false"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "datastream_stream_postgresql"
+        pull_external: true
+        primary_resource_id: "default"
+        skip_vcr: true
```

Comment:
in https://github.com/GoogleCloudPlatform/magic-modules/pull/7115#discussion_r1084433228 I meant remove the `skip_vcr` lines, since these examples do not need to be skipped. Additionally, we can remove the `pull_external` lines.
This will enable the tests to run in our CI.

Reply:
done